src/pages/Login/index.js

```diff
@@ -1,45 +1,72 @@
-import React, { Component } from 'react'
-import { Redirect } from 'react-router-dom'
-import { connect } from 'react-redux'
+import React, { Component } from 'react';
+import uuidValidate from 'uuid-validate';
+import { has } from 'ramda';
 import { message } from 'antd';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from "redux";
 
-import * as R from 'ramda'
-
-import LoginContainer from './LoginContainer'
-import uuidValidate from 'uuid-validate'
+import LoginContainer from '../../containers/Login';
+import { signIn } from '../../services/auth';
+import { onSubmit } from './LoginRedux/action'
 
 class LoginPage extends Component {
-  hasAuth = R.has('auth')
-  hasToken = R.has('token')
+  state = {
+    authenticated: false,
+  }
 
-  success = () => {
-    message.success('Bem-vindo ao Estoque')
+  verifyAuth = () => {
+
+    if(has('auth', this.props) && has('token', this.props.auth)){
+      return this.setState({authenticated : uuidValidate(this.props.auth.token)})
+    }
   }
 
-  render() {
+  success = () => {
+    message.success('Bem-vindo ao Estoque');
+  };
+
+  error = () => {
+    message.error("Inautorizado, confira username e senha por favor");
+  };
+
+  handleSubmit = async ({ username, password }) => {
+    const { status, data } = await signIn({ username, password });
+
+    switch (status) {
+      case 200:
+        await this.props.onSubmit(data);
+        this.verifyAuth()
```

Comment:
Para nao termos essa verificação na mão podemos usar o try catch o código completo para o handle submit ficaria assim
```suggestion
  handleSubmit = async ({ username, password }) => {
    try {
      const { status, data } = await signIn({
        username,
        password,
        typeAccount: {
          stock: true
        }
      });

      if (Number(status) === 401) {
        throw new Error('User UNAUTHORIZED')
      };

      await this.props.onSubmit(data);
      this.verifyAuth();
      this.state.authenticated && this.success();
    } catch (error) {
      this.error();
    }
  };
```